api/src/aggregate/aggregate.service.ts

```diff
@@ -1,20 +1,30 @@
 import { Injectable } from '@nestjs/common'
 import { Prisma } from '@prisma/client'
 import { PrismaService } from '../prisma.service'
+import { RedisService } from '../redis.service'
 import { AggregateDto } from './dto/aggregate.dto'
-import { LatestAggregateDto } from './dto/latest-aggregate.dto'
+import { LatestAggregateByIdDto, LatestAggregateDto } from './dto/latest-aggregate.dto'
 
 @Injectable()
 export class AggregateService {
-  constructor(private prisma: PrismaService) {}
+  constructor(private prisma: PrismaService, private readonly redis: RedisService) {}
 
   async create(aggregateDto: AggregateDto) {
+    const _timestamp = new Date(aggregateDto.timestamp)
     const data: Prisma.AggregateUncheckedCreateInput = {
-      timestamp: new Date(aggregateDto.timestamp),
+      timestamp: _timestamp,
       value: aggregateDto.value,
       aggregatorId: BigInt(aggregateDto.aggregatorId)
     }
 
+    await this.redis.set(
+      `latestAggregate:${BigInt(aggregateDto.aggregatorId).toString()}`,
```

Comment:
Can you please make an issue for changing `aggregatorId` type to string right after it is loaded from DB. Currently, we have to unnecessarily serialize it before passing it to this function and then we deserialize, however, we use string eventually only. 🤔

Reply:
issue has been created
https://github.com/Bisonai/orakl/issues/1016